modules/shortcodes/instagram.php

```diff
@@ -161,8 +163,49 @@ function jetpack_instagram_handler( $matches, $atts, $url ) {
 		$url_args['hidecaption'] = 'true';
 	}
 
-	$url = esc_url_raw( add_query_arg( $url_args, 'https://api.instagram.com/oembed/' ) );
+	$use_cache     = jetpack_instagram_use_cache( $matches, $atts, $passed_url );
+	$cache_key     = 'oembed_response_body_' . md5( add_query_arg( $url_args, 'https://api.instagram.com/oembed/' ) );
```

Comment:
Should we add the `esc_url_raw` back in here, so the cache key remains the same as in previous versions?

Reply:
I'm not sure. I pulled it out because I think it makes it slightly more readable and the existing cached values have a relatively short cache ttl anyways. If you think it's necessary, I can add it back in.